kubemarine/__main__.py

```diff
@@ -115,6 +115,10 @@
         'description': "Print current release version",
         'group': 'other'
     },
+    'config': {
+        'description': "Print supported configurations of 3rd-parties with versions compatibility",
```

Comment:
```suggestion
        'description': "Print supported configurations of 3rd-parties with versions compatibility",
```
```suggestion
        'description': "Print supported k8s versions with respective configurations of 3rd-parties.",
```
@shmo1218  pls comment

Reply:
I used more generic `configurations of 3rd-parties` and moved information about Kubernetes inside "kubernetes" subsection. See configuration format in **Solution**. This is to make introducing of more compatibilities (helm, ansible?) with probably no breaking changes. If we aim this procedure only for `k8s versions`, such nested section is redundant.

Reply:
Hi, @koryaga,

"Print the supported k8s versions with the respective configurations of third-parties."